stubs/toml/toml/encoder.pyi

```diff
@@ -17,8 +17,8 @@ class TomlEncoder(Generic[_MappingT]):
     def __init__(self: TomlEncoder[dict[str, Any]], _dict: type[dict[str, Any]] = ..., preserve: bool = ...) -> None: ...
     def get_empty_table(self) -> _MappingT: ...
     def dump_list(self, v: Iterable[object]) -> str: ...
-    def dump_inline_table(self, section: dict[str, Any] | Any) -> str: ...
-    def dump_value(self, v: Any) -> str: ...
+    def dump_inline_table(self, section: object) -> str: ...
+    def dump_value(self, v: object) -> str: ...
```

Comment:
From what I can see of the implementation, and from trying it, if it doesn't specifically handle an object it just stringifies it.

Reply:
While technically it can handle all objects, in that it doesn't throw an exception, not all objects are correct arguments. `"<object object at 0x000001FF99FF85B0>"` is obviously not useful as an output.

Reply:
And trying to represent intended values, we kindof hit the same issue as JSON.
There's also classes with custom __repr__ that can be dumped.